src/dotnet/commands/dotnet-help/HelpCommand.cs

```diff
@@ -33,7 +33,8 @@ public class HelpCommand
   run           Compiles and immediately executes a .NET project
   test          Runs unit tests using the test runner specified in the project
   pack          Creates a NuGet package
-  vstest        Runs tests from the specified files";
+  vstest        Runs tests from the specified files
+  add           Group of commands - run `dotnet add --help` for more information";
```

Comment:
@krwq we will introduce grouping in this text in #4763. I suggest we mark the CRUD commands as such clearly in their own group. The group name should be "Project modification commands". Also, these commands should go above the `vestest` and `msbuild` ones, since they are pretty likely to be used more extensively than those.